package.json

```diff
@@ -22,22 +22,21 @@
     "test": "npm run test:coverage",
     "test:coverage": "BABEL_ENV=test nyc npm run test:unit",
     "test:html-coverage": "BABEL_ENV=test nyc --reporter=html npm run test:unit",
-    "test:unit": "BABEL_ENV=test ava test/unit/** --verbose",
-    "test:unit:watch": "BABEL_ENV=test ava test/unit/** --verbose --watch",
+    "test:unit": "BABEL_ENV=test jest test/unit/** --verbose",
+    "test:unit:watch": "BABEL_ENV=test jest test/unit/** --verbose --watch",
     "config:restore": "test/integration/scripts/post-test.sh",
     "test:e2e": "npm run test:e2e:setup && npm run test:e2e:run && npm run config:restore",
     "test:e2e:fresh": "npm run test:e2e:setup && npm run test:e2e:update && npm run config:restore",
     "test:e2e:setup": "test/integration/scripts/pre-test.sh",
-    "test:e2e:run": "BABEL_ENV=test ava test/integration/cmds/** -c 1 --verbose",
-    "test:e2e:update": "BABEL_ENV=test ava test/integration/cmds/** --verbose --update-snapshots",
-    "test:e2e:watch": "BABEL_ENV=test ava test/integration/cmds/** --verbose --watch",
+    "test:e2e:run": "BABEL_ENV=test jest test/integration/cmds/** --verbose --runInBand",
```

Comment:
`--runInBand` is necessary for tests to run on travis without timing out, but not necessary locally (and really slows things down, from ~85secs to >180secs).
WDYT about adding a command for running e2e tests locally that doesnt have this flag? especially since we are going to rethink the script structure soon anyway?

Reply:
let's keep `test:e2e:run` for local test and add `test:e2e:run:ci` with `--runInBand`